platforms/hyperledger-fabric/configuration/chaincode-install-instantiate.yaml

```diff
@@ -52,16 +52,17 @@
       loop: "{{ network['channels'] }}"
       when: add_new_org == 'false' and '2.' in network.version    
         
-    # This task invokes chaincode after the chaincode installation for the new organization to 
-    # be added in existing network
+    # This task invokes chaincode after the chaincode has been commited in v2 and
+    # after chaincode installation for the new organization to be added in existing network
     - name: Invoke chaincode
       include_role:
         name: "create/chaincode/invoke"
       vars:
         participants: "{{ item.participants }}"
         docker_url: "{{ network.docker.url }}"
+        approvers: "{{ item.endorsers }}"
       loop: "{{ network['channels'] }}"
-      when: add_new_org == 'true'
+      when: add_new_org == 'false' and '2.' in network.version    
```

Comment:
Invoke is also required after the chaincode installation ( and approve in case of fabric 2.2.0 ) for adding the new organization. The check should be `add_new_org == 'true' or '2.' in network.version`